src/controllers/midi/midicontroller.cpp

```diff
@@ -288,7 +288,7 @@ void MidiController::processInputMapping(const MidiInputMapping& mapping,
             return;
         }
 
-        QScriptValue function = pEngine->resolveFunction(mapping.control.item);
+        QScriptValue function = pEngine->wrapFunctionCode(mapping.control.item, 5);
```

Comment:
That works -- though it creates more complexity / tight coupling between the controller and the script engine. We could create an opaque reference/token or something.

Unless @Be-ing wants to go for "extra credit" here from my POV this branch is a strict improvement over master though.